packages/client/src/components/addReaction/addReaction.tsx

```diff
@@ -9,12 +9,13 @@ interface AddReactionProps {
   emojiSet: Nullable<EmojiType[]>
   reactions?: Nullable<ReactionType[]>
   commentId: number
+  userId?: number
   onAddRection: () => void
 }
-export const AddReaction = ({ reactions, emojiSet, commentId, onAddRection }: AddReactionProps) => {
+export const AddReaction = ({ reactions, emojiSet, commentId, onAddRection, userId }: AddReactionProps) => {
   const [isEmojiVisible, setEmojiVisibility] = useState(false)
 
-  return (
+  return userId ? (
```

Comment:
Для таких случаев удобно делать досрочный выход, чтоб не увеличивать вложенность
```suggestion
  if(!userId) return <div />
  
  return (
    <div className={styles.reactionActsContainer}>
    ...
  
```
Ну и ещё появляется мысль, что раз без `userId` компонент нельзя использовать, то возможно стоит сделать пропс `userId` обязательным и проверять наличие `userId` на уровне, где используется `AddReaction`, если его нет, то и отображать `AddReaction` бессмыслено